src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIResponse.java

```diff
@@ -98,7 +105,27 @@ public int getHttpStatus() {
   }
 
   public boolean isSuccess() {
-    return successCodes.contains(getHttpStatus());
+    boolean isSuccess = false;
+    // ServiceNow Rest API may return 200 OK response with an error message in the response body.
+    // Normally we would expect non-200 response code if there's an error.
+    if (getRetryable()) {
```

Comment:
no need to call the getter, just access isRetryable directly. Similar comment elsewhere (getResponseBody()).

Reply:
Removed the getter : `getRetryable()` 2c4538f24a45de8576bb7a51b2422ad070a696dd

`getResponseBody()` is being used in several places outside the`RestAPIResponse` class